asab/library/service.py

```diff
@@ -260,33 +252,27 @@ async def _list(self, path, tenant, providers):
 				continue
 
 			for item in ress:
-
 				item.disabled = self.check_disabled(item.name, tenant=tenant)
 
-				# If the item already exists, merge it
-				pitem = uniq.get(item.name)
-				if pitem is not None:
+				# If the item already exists, merge or override it
+				if item.name in uniq:
+					pitem = uniq[item.name]
```

Comment:
Keep `pitem = uniq.get(item.name)` and test for a None ... it does the same with the half of the cost.